### DISCUSSION TITLE
No output when using run_inference.py

### DISCUSSION THREAD

**Discussion by phalcomb** [microsoft/BitNet#148] [Q&A]
Hi all, looking for a little guidance here. I built a brand-new test VM with Ubuntu 22.04, just the basic server. Installed all the prereqs per the instructions in the README.md file, everything builds fine, I can see it was compiled with clang 18. However, when I run run_inference.py I get blank output using the model described in the README.md file. I've tried using other models, some have spit out gibberish, another just output hashtags. Any ideas?

**Comment by castortech** [comment]
Exact same result here.
